app/Controllers/Contact.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace App\Controllers;
+
+use App\Helpers\EmailHelper;
+use CodeIgniter\HTTP\ResponseInterface;
+
+class Contact extends BaseController
+{
+    private const CONTACT_RULES = [
+        'name' => 'required',
+        'email' => 'required|valid_email',
+        'subject' => 'required|max_length[78]',
+        'message' => 'required|max_length[2000]',
```

Comment:
I dont validate the length in the frontend explicit because html should do that just fine.